### PR TITLE
[Validator] Updated missing Latvian translation keys

### PR DESCRIPTION
**Pull request by thunderer**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | #51946 
| License       | MIT

Updated missing Latvian translations with help from a Latvian.

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf [hunk 1]
```diff
@@ -402,6 +402,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Tīkla maskas (netmask) vērtībai jābūt starp {{ min }} un {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Faila nosaukums ir pārāk garš. Tas var būt { filename_max_length }} rakstzīmi vai īsāks.|Faila nosaukums ir pārāk garš. Tajā var būt {{ filename_max_length }} rakstzīmes vai īsāks.</target>
```
Comment (k0d3r1s) [inline src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf]: missing {  
also "rakstzīme" not "rakstzīmi"  
also for consistency - "Tajā var būt" should be replaced by "Tas var būt"

File: src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf [hunk 2]
```diff
@@ -402,6 +402,30 @@
...
+                <source>Using invisible characters is not allowed.</source>
+                <target>Neredzamu rakstzīmju izmantošana nav atļauta.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Mixing numbers from different scripts is not allowed.</source>
+                <target>Nav atļauts sajaukt numurus no dažādiem skriptiem.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Using hidden overlay characters is not allowed.</source>
+                <target>Slēpto pārklājuma rakstzīmju izmantošana nav atļauta.</target>
```
Comment (k0d3r1s) [inline src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf]: "Slēptu" would be more appropriate here

File: src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf [hunk 3]
```diff
@@ -402,6 +402,30 @@
...
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Faila nosaukums ir pārāk garš. Tas var būt { filename_max_length }} rakstzīmi vai īsāks.|Faila nosaukums ir pārāk garš. Tajā var būt {{ filename_max_length }} rakstzīmes vai īsāks.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>Paroles stiprums ir pārāk zems. Lūdzu, izmantojiet spēcīgāku paroli.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Šajā vērtībā ir rakstzīmes, kuras nav atļautas pašreizējā ierobežojuma līmenī.</target>
```
Comment (k0d3r1s) [inline src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf]: "Šajā vērtībā ir rakstzīmes" should be "Šī vērtība satur rakstzīmes" for closer translation of origina value

**Comment by thunderer** [comment]
@k0d3r1s thank you, I applied all your suggestions! Could you please re-review?

**Review by k0d3r1s — APPROVED** [review]
LGTM

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @thunderer.

**Pull request merged**